README.md

```diff
@@ -9,13 +9,23 @@
 - PHP >= 7.1  (extension requirements in composer.json).
 
 ## Installation
-Use [composer](https://getcomposer.org/?target=_blank) to install the package
+The sdk uses the [httplug](http://docs.php-http.org/en/latest/httplug/introduction.html) library for the http client implementation, 
+so you will need to install a library that provides the virtual `php-http/client-implementation` composer package, e.g. `php-http/guzzle6-adapter`. 
+
+Use [composer](https://getcomposer.org/?target=_blank) to install a http client of your choice 
+
+```bash
+$ composer require php-http/guzzle6-adapter
+```
+
+and the sdk itself:
+
 ```bash
 $ composer require starweb/shop-api-sdk
 ```
 
 # Documentation
-The Starweb Shop API SDK uses an auto generated client 
+The Starweb Shop API SDK uses an auto generated client based on [Jane](https://jane.readthedocs.io/) at its core.
```

Comment:
I think we also should add something here of how to use it:

```
$credentials = new ClientCredentials('my-client-id', 'my-client-secret');
$starweb = Starweb::create($credentials, 'https://my-shop.starwebserver.se/api/v2');
$client = $starweb->getClient();

$products = $client->listProducts(); // fetch products
```

Reply:
This would have saved me a LOT of time when I tried to get started 😉

Reply:
added some documentation how to get started